<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>288</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="topBtnLayout">
     <item>
      <widget class="QPushButton" name="loadMrs">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Load MRS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createMrs">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Create MRS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="monitorDir">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Monitor DIR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveMrs">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Save MRS</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="magicBox">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#magicBox {
	color: rgb(0, 0, 150);
	font-weight: bold;
}</string>
     </property>
     <property name="title">
      <string>Magic</string>
     </property>
     <layout class="QGridLayout" name="magicLayout" rowstretch="1,1,1,1,0">
      <item row="2" column="1">
       <widget class="QLineEdit" name="magicName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="magicIdx">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="magicMpCost">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="magicNameLbl">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Magic Name:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="magicBodyMovementLbl">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Body Movement:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="magicDefensePowerLbl">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Defense Power:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="magicHpLbl">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Hp:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="magicMpCostLbl">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Mp Cost:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="magicIdxLbl">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Index:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QSpinBox" name="magicMp">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <number>-32768</number>
        </property>
        <property name="maximum">
         <number>32767</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="magicDescriptionLbl">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Magic Description:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="magicType">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Attack</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Enhance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Recovery</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Assist</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Special</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="magicTypeLbl">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Type:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="magicAttackPower">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="magicHp">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <number>-32768</number>
        </property>
        <property name="maximum">
         <number>32767</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="8">
       <widget class="QGroupBox" name="magicAttrBox">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#magicAttrBox {
	color: rgb(0, 0, 150);
}</string>
        </property>
        <property name="title">
         <string>Magic Attribute</string>
        </property>
        <layout class="QHBoxLayout" name="magicAttrHorLayout">
         <item>
          <widget class="QCheckBox" name="magicPoisonChk">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>9</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Poison</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="magicMessChk">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>9</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Mess</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="magicSealChk">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>9</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Seal</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="magicSleepChk">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>9</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Sleep</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="magicGroupEffectChk">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>9</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Group Effect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="magicEffectiveRoundLbl">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>9</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Effective Round:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="magicEffectiveRound">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>9</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>9</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="magicMpLbl">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Mp:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QSpinBox" name="magicBodyMovement">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="magicAttackPowerLbl">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Attack Power:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QSpinBox" name="magicDefensePower">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="7">
       <widget class="QTextEdit" name="magicDescription">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="magicEffectsLbl">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{ color: rgb(0, 0, 150); }</string>
        </property>
        <property name="text">
         <string>Effects:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QSpinBox" name="magicEffects">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>loadMrs</tabstop>
  <tabstop>createMrs</tabstop>
  <tabstop>monitorDir</tabstop>
  <tabstop>saveMrs</tabstop>
  <tabstop>magicType</tabstop>
  <tabstop>magicIdx</tabstop>
  <tabstop>magicName</tabstop>
  <tabstop>magicHp</tabstop>
  <tabstop>magicMp</tabstop>
  <tabstop>magicAttackPower</tabstop>
  <tabstop>magicDefensePower</tabstop>
  <tabstop>magicBodyMovement</tabstop>
  <tabstop>magicMpCost</tabstop>
  <tabstop>magicEffects</tabstop>
  <tabstop>magicPoisonChk</tabstop>
  <tabstop>magicMessChk</tabstop>
  <tabstop>magicSealChk</tabstop>
  <tabstop>magicSleepChk</tabstop>
  <tabstop>magicGroupEffectChk</tabstop>
  <tabstop>magicEffectiveRound</tabstop>
  <tabstop>magicDescription</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
